docs/release-notes/relnotes-2111-1.md

```diff
@@ -0,0 +1,32 @@
+---
```

Comment:
Why do we need a the relnotes-2111-1.md?  Is the currentRelease.md sufficient?

Reply:
This file is linked in the release [here](https://github.com/pingidentity/pingidentity-docker-builds/releases/tag/2111.1). currentRelease.md is also the same file, we're just linking them exactly to the release as well.